### PR TITLE
Drop 'surfconext-informed' support

### PR DESCRIPTION
**Pull request by MKodde**
For now we do not use this status. More research needs to be performed
in order to integrate it adequately in the process. So for now the
status is removed.

~~A migration was added to update the existing services with that status
to 'requested'.~~ 
**[Edit Feb 18 16:26](https://github.com/SURFnet/sp-dashboard/pull/235#issuecomment-464773971)** *See comment below why this migration was dropped*

https://www.pivotaltracker.com/story/show/164008419

### CODE REVIEW

**Comment by pablothedude** [comment]
I saw `yarn audit` failed. Could `failonerror` be disabled, or could you fix the error? If you have another solution feel free to do so.

I also saw that the jest service form mock data had this radio option as possible option. I don't mind if you keep it there but I wanted you to know that it still was there.

File: src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php [hunk 1]
```diff
@@ -50,7 +50,6 @@ class Service
     // Production connection status: (not-requested|requested|active|surfconext-informed)
```
[the remark this answers is not in the document]
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php]: Wel grepped! Removing it.

**Comment by MKodde** [comment]
After discussion, we decided to remove the migration that was previously on this commit. Updating existing services with the informed state will be updated manually. As there is no easy way to set the correct status. This would involve querying manage for each service and determine on a per result basis what the service status should be.

**Comment by MKodde** [comment]
The build breaks on Jest. Typescript support for Jest is not yet bumped to support Jest 24. Jest 23 yields the security warnings. 

For now we can ignore them in my humble opinion. The test tools are dev dependencies only. And should not cause any problems on production builds.

**Comment by MKodde** [comment]
Merging: also see #239

**Pull request merged**